lib/src/rules/avoid_implementing_value_types.dart

```diff
@@ -0,0 +1,122 @@
+// Copyright (c) 2018, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'package:analyzer/analyzer.dart';
```

Comment:
Please replace this import with imports of more specific libraries. (This library will be deprecated in the fairly near future.)